config/routes.rb

```diff
@@ -2,7 +2,8 @@
   devise_for :admin_users, ActiveAdmin::Devise.config
   ActiveAdmin.routes(self)
   devise_for :users
-  root to: 'pages#home'
+  get '/', to: redirect('/categories/47')
```

Comment:
instead of doing a redirect, we could load this category on the index method and render the show view